internal/pretty/pretty.go

```diff
@@ -36,17 +35,17 @@ const jsonIndent = "  "
 // If marshal fails, it falls back to fmt.Sprintf("%+v").
 func ToJSON(e any) string {
```

Comment:
@dfawley, I think we should still support both versions to keep backwards compatibility to older proto messages?